docs/UserGuide.md

```diff
@@ -37,38 +37,44 @@ If you are an experienced user, we have provided a convenient [Summary of Comman
-  * [**4.7** `delete`Delete client contact](#delete-delete-client-contact)
-  * [**4.8** `batch` Execute commands in batch](#batch-execute-commands-in-batch)
-  * [**4.9** `sort`: Sort list of clients](#sort-sort-list-of-clients)
-  * [**4.10** `meet`: Schedule a meeting with a client](#meet-schedule-a-meeting-with-a-client)
-  * [**4.11** `addshortcut`: Add shortcut](#addshortcut-add-shortcut)
-  * [**4.12** `editshortcut`: Edit shortcut](#editshortcut-edit-shortcut)
-  * [**4.13** `deleteshortcut`: Delete shortcut](#deleteshortcut-delete-shortcut)
-  * [**4.14** `listshortcut`: List all shortcuts](#listshortcut-list-all-shortcuts)
-  * [**4.15** `clearshortcut`: Clear all shortcuts](#clearshortcut-clear-all-shortcuts)
-  * [**4.16** `lock`: Lock ClientBook with a user-selected password](#lock-lock-clientbook-with-a-user-selected-password)
-  * [**4.17** `unlock`: Unlock ClientBook](#unlock-unlock-clientbook)
-  * [**4.18** `exit`: Exiting the program](#exit-exiting-the-program)
-  * [**4.19** Saving data](#saving-data)
-  * [**4.20** Editing data file](#editing-data-file)
-  * [**4.21** Upcoming features!](#upcoming-features-v20-and-beyond)
+    * [<span style="color:#b573c9">**4.1** Contact Management</span>](#span-stylecolorb573c9contact-management-featuresspan)
+        * [**4.1.1** `add`: Add client contact](#add-add-client-contact)
+        * [**4.1.2** `edit`: Edit client contact](#edit-edit-client-contact)
+        * [**4.1.3** `delete`Delete client contact](#delete-delete-client-contact)
+        * [**4.1.4** `batch` Execute commands in batch](#batch-execute-commands-in-batch)
+        * [**4.1.5** `meet`: Schedule a meeting with a client](#meet-schedule-a-meeting-with-a-client)
+    * [<span style="color:#f75c66">**4.2** Contact Viewing</span>](#span-stylecolorf75c66contact-viewing-featuresspan)
+        * [**4.2.1** `list`: List all clients](#list-list-all-clients)
+        * [**4.2.2** `find`: Search for client contact based on keywords](#find-search-for-client-contact-based-on-keywords)
+        * [**4.2.3** `policy`: Display policies associated with a client](#policy-display-policies-associated-with-a-client)
+        * [**4.2.4** `sort`: Sort list of clients](#sort-sort-list-of-clients)
+    * [<span style="color:#ebc000">**4.3** Shortcuts</span>](#span-stylecolorebc000shortcutsspan) 
+        * [**4.3.1** `addshortcut`: Add shortcut](#addshortcut-add-shortcut)
+        * [**4.3.2** `editshortcut`: Edit shortcut](#editshortcut-edit-shortcut)
+        * [**4.3.3** `deleteshortcut`: Delete shortcut](#deleteshortcut-delete-shortcut)
+        * [**4.3.4** `listshortcut`: List all shortcuts](#listshortcut-list-all-shortcuts)
+        * [**4.3.5** `clearshortcut`: Clear all shortcuts](#clearshortcut-delete-all-shortcuts)
+    * [<span style="color:#3cb44b">**4.4** General</span>](#span-stylecolor3cb44bgeneral-featuresspan)  
```

Comment:
The links for the colored words do not work.

Reply:
Fixed.